README.md

```diff
@@ -1,31 +1,37 @@
 # xmtp-react-native
 
-![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre--Preview-red)
+![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Beta-yellow)
 
 This repo provides a package you can use to build with XMTP in a React Native or Expo app.
 
-> **Important:**  
-> This SDK is in **Pre-Preview** status and ready for you to start experimenting with.
+> **Important**  
+> This SDK is in **beta** status and ready for you to start experimenting with.
 >
-> However, we do **not** recommend using Pre-Preview software in production apps. Software in this status will change as we add features and iterate based on feedback.
+> However, we do **not** recommend using beta software in production apps. Software in this status will change as we add features and iterate based on feedback.
 
 We're still working on adding several features to bring this SDK to parity with the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (`xmtp-js`). Here's a [list of features and issues](https://github.com/xmtp/xmtp-react-native/issues/14) we're working on.
 
 Is there a feature you need that's not on the list? Please [open an issue](https://github.com/xmtp/xmtp-react-native/issues).
 
 Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1042)!
 
-To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+To learn more about XMTP and get answers to frequently asked questions, see [XMTP documentation](https://xmtp.org/docs).
 
 ![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
 
-## Example app
+## Reference docs
 
-This repo provides an [example app](example) that you can use to experiment and explore implementation details.
+You can use the `xmtp-js` client SDK [reference documentation](https://xmtp-js.pages.dev/modules) as reference documentation for this SDK.
+
+## Quickstart app
+
+<!--Per our offsite discussion, I think this is a quickstart app, not an example app? If we make this change, I think we need to change the directory name.-->
```

Comment:
See comment

Reply:
@jhaaaa I talked again with @nakajima and I think on second thought these should actually stay as example. I don't think any word is super great but I think example is the most clear that it shouldn't be copy and pasted directly into their app and more is an example of what could be done. Maybe a note about database first application or a link to our RN playground for a more full featured reference could suffice.